docs/layers.md

```diff
@@ -732,6 +732,7 @@ Places with `kind` values of `continent`, `country`, with others added starting
 * `id`: The `osm_id` from OpenStreetMap or Natural Earth id
 * `kind`: normalized values between OpenStreetMap and Natural Earth
 * `population`: population integer values from OpenStreetMap or Natural Earth's maximum population value.
+* `population_rank`: A value from 18 down to 0, indicating how large the population is on a particular place. A larger value indicates a bigger population.
```

Comment:
Please add full documentation for the class ranges in a new sub-section here (we do this for various kind_detail enumerations, for example).

Reply:
https://github.com/tilezen/vector-datasource/issues/988#issuecomment-453262892

```
18: Over 1b
17: 100m to 1b
16: 50m to 100m
15: 20m to 50m
14: 10m to 20m
13: 5m to 10m
12: 1m to 5m
11: 500k to 1m
10: 200k to 500k
9: 100k to 200k
8: 50k to 100k
7: 20k to 50k
6: 10k to 20k
5: 5k to 10k
4: 2k to 5k
3: 1k to 2k
2: 200 to 1k
1: Less than 200
0: Locale (no population) / places that only are visible at 50k scale or larger (but not neighbourhoods)
```

Reply:
Added in 06b65aa.